### DISCUSSION TITLE
为什么我在Windows下编译出来很卡顿

### DISCUSSION THREAD

**Discussion by mogoweb** [xland/ScreenCapture#56] [Q&A]
我使用的是 Visual Studio Community 2022，Windows SDk 版本位 10.0.20348.0
SKIA 库是我自己编译出来的，版本为 chrome/m122，生成skia库的参数为 `bin\gn gen out\x64 --args="skia_use_gl=true is_official_build=false is_component_build=false win_sdk=\"C:\\Program Files (x86)\\Windows Kits\\10\" win_sdk_version=\"10.0.20348.0\" win_vc=\"D:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\" is_debug=false"`
不知道为什么生成出来的ScreenCapture没有您在Release发布的流畅。请问您是用的什么样的 skia 库？

**Comment by xland** [comment]
我的编译参数：
```
bin/gn gen out/release --args='clang_win="C:\Program Files\LLVM" cc="clang" cxx="clang++" extra_cflags=["/MT"] is_debug=false is_official_build=true skia_use_system_expat=false skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_system_harfbuzz=false skia_use_system_icu=false skia_enable_skparagraph=true skia_enable_skshaper=true skia_enable_skunicode=true'
ninja -C out/release
```

**Reply by mogoweb** [reply]
感谢！按照您的参数重新编译skia，问题解决
